widget/scroller.go

```diff
@@ -354,6 +354,24 @@ func (s *ScrollContainer) CreateRenderer() fyne.WidgetRenderer {
 	return scr
 }
 
+
+//ScrollToBottom will scroll content to container bottom - to show latest info which end user just added
+func (s *ScrollContainer) ScrollToBottom(isNeedRefresh bool) {
+	s.Offset.Y = s.Content.Size().Height - s.Size().Height + s.MinSize().Height
+	if isNeedRefresh {
+		s.Refresh()
+	}
+}
+
+//ScrollToTop will scroll content to container top. It is easy if compare to ScrollToBottom
+func (s *ScrollContainer) ScrollToTop(isNeedRefresh bool) {
+	s.Offset.Y = 0
+	if isNeedRefresh {
+		s.Refresh()
+	}
+}
+
+
```

Comment:
I think this extra line is why Travis is failing - but there also might be trailing whitespace above (see travis errors for info).

Reply:
Thanks for point out this.